packages/selinux-policy/processes.cil

```diff
@@ -23,6 +23,6 @@
 (classmapping processes interact (
   process (not (
     dyntransition transition setcurrent setexec
-    setfscreate setkeycreate setsockcreate
+    setfscreate setkeycreate setsockcreate noatsecure
```

Comment:
Conceptually this could also go into the `relax` class mapping, which would leverage the current rule that bans host programs from using any of those process actions.